CalculatorKata/Program.cs

```diff
@@ -1,25 +1,39 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace CalculatorKata
 {
     public class Program
     {
         static void Main(string[] args)
         {
-           
+            
         }
-        public int add(String numbers)
+        public  int add(String numbers)
         {
             if (string.IsNullOrWhiteSpace(numbers))
                 return 0;
             char[] delimiterChars = { '\n', ',', ';', '/' };
             string[] num = numbers.Split(delimiterChars, System.StringSplitOptions.RemoveEmptyEntries);
             int sum = 0;
+            List<int> negativenum = new List<int>();
             for (int i = 0; i < num.Length; i++)
-            {
+            {   if(Convert.ToInt32(num[i]) < 0)
+                {
+                    negativenum.Add(Convert.ToInt32(num[i]));
+                    
+                }
                 sum += Convert.ToInt32(num[i]);
```

Comment:
Each number is converted twice, once in the if statement and once when adding it to the sum, how could we change to call the convert only once?